fees/maverick.ts

```diff
@@ -19,6 +19,13 @@ const adapter: SimpleAdapter = {
         methodology,
       },
     },
+    [CHAIN.ERA]: {
+      fetch: fetchFee(CHAIN.ERA),
+      start: async () => 1337265,
```

Comment:
I checked subgraph there is a volume since 1681257600

Reply:
fixed